Changes to the case feature in 8.7 (#3053)
elastic/security-docs: docs/cases/cases-manage.asciidoc

[role="screenshot"]
image::images/cases-open-vis.png[Shows where the Open Visualization option is]

[float]
[[cases-copy-case-uuid]]

> === Copy a case's UUID

Author: ```suggestion
=== Copy the case UUID
```

> Each case has a universally unique identifier (UUID) that you can copy and share. To copy a case's UUID to a clipboard, go to the Cases page and select *Actions* -> *Copy Case ID* for the case you want to share. Alternatively, go to a case's details page, then select *More actions* menu (…​) -> *Copy Case ID*.

Reviewer: ```suggestion
Each case has a universally unique identifier (UUID) that you can copy and share. To copy a case's UUID to a clipboard, go to the Cases page and select *Actions* -> *Copy Case ID* for the case you want to share. Alternatively, go to a case's details page, then from the *More actions* menu (…​), select *Copy Case ID*.
```

> Each case has a universally unique identifier (UUID) that you can copy and share. To copy a case's UUID to a clipboard, go to the Cases page and find the case you want to share, click the *Actions* menu, then select *Copy Case ID*. Alternatively, go to a case's details page, click the *More actions* menu (…​), and select *Copy Case ID*.

Reviewer 2: Optional streamlining:
```suggestion
Each case has a universally unique identifier (UUID) that you can copy and share. To copy a case's UUID to a clipboard, go to the Cases page and select *Actions* -> *Copy Case ID* for the case you want to share. Alternatively, go to a case's details page, then select *More actions* menu (…​) -> *Copy Case ID*.
```

[role="screenshot"]

> image::images/cases-copy-case-id.png[Shows where the Copy Case ID option is 60%,60%]

Reviewer 2: ```suggestion
image::images/cases-copy-case-id.png[Copy Case ID option in More actions menu 60%,60%]
```
